scripts/modules/elastic_stack.py

```diff
@@ -656,6 +656,11 @@ def __init__(self, **options):
                 self.environment["STATUS_ALLOWANONYMOUS"] = "true"
         self.environment["ELASTICSEARCH_URL"] = ",".join(self.options.get(
             "kibana_elasticsearch_urls") or [self.DEFAULT_ELASTICSEARCH_HOSTS])
+        if self.at_least_version("7.6") and options.get("xpack_secure"):
+            self.environment["XPACK_SECURITY_LOGINASSISTANCEMESSAGE"] = "Login\_details:\_`{}/{}`.\_" \
```

Comment:
As I already described in the PR description, there is no other way to customize the login page using the env variables with docker-compose. That's the reason for using the `_` as a word separator.

Reply:
For people who come across this in the future, [this is the PR](https://github.com/docker/compose/pull/6923) that will eventually fix this in Docker Compose.